config/default/manager_image_patch.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: apps/v1
-kind: StatefulSet
+kind: Deployment
 metadata:
   name: controller-manager
   namespace: system
 spec:
   template:
     spec:
       containers:
-      - image: gcr.io/k8s-staging-cluster-api/cluster-api-controller:latest
+      - image: gcr.io/k8s-staging-cluster-api/cluster-api-controller:master
```

Comment:
This needs to be pushed, but I figured we should change it now so we can start building on this tag. We can build manually for now and automate it once things are in place

Reply:
I think we should stick with `latest`, no need to introduce the use of a loaded word like `master` if we do not need to.

Reply:
Isn't the `latest` tag behavior to always store the actual latest pushed version? I figured we might publish a v1alpha1 version at some point and it'll overwrite the `latest` tag, which could break some users.

Reply:
I wouldn't expect anyone to be using `latest`, especially since we use `dev` by default for dev workflows. I would only expect latest to be used for developing against master. If we need to introduce a `latest` equivalent for a branch, we can always introduce `branch-latest`

Reply:
sounds good, reverted